src/test/java/com/wikia/webdriver/testcases/adstests/TestAdFliteSkinPresence.java

```diff
@@ -0,0 +1,84 @@
+package com.wikia.webdriver.testcases.adstests;
+
+import com.wikia.webdriver.common.dataprovider.ads.AdsDataProvider;
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+import com.wikia.webdriver.common.templates.TemplateNoFirstLoad;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.AdsBaseObject;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Dimension;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.FindBy;
+import org.testng.annotations.Test;
+
+import java.io.IOException;
+import java.util.List;
+
+public class TestAdFliteSkinPresence extends TemplateNoFirstLoad {
+
+  @Test(
+      dataProviderClass = AdsDataProvider.class,
+      dataProvider = "fliteSkin",
+      groups = "TestSkinPresence_GeoEdgeFree"
+  )
+  public void TestAdFliteSkinPresence(String wikiName, String article,
+                                      Dimension windowResolution) throws IOException {
+    String testedPage = urlBuilder.getUrlForPath(wikiName, article);
+    PageObjectLogging.log("Window resolution: ", String.valueOf(windowResolution.width), true);
+    AdsBaseObject adsBaseObject = new AdsBaseObject(driver, testedPage, windowResolution);
+    adsBaseObject.verifySkin("src/test/resources/adsResources/flite_skin_left_1.png",
+                             "src/test/resources/adsResources/flite_skin_right_1.png",
+                             null,
+                             null);
+    driver.switchTo().frame("google_ads_iframe_/5441/wka.life/_adtest//article/gpt/TOP_LEADERBOARD_0");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "google_ads_iframe_/5441/wka.life/_adtest//article/gpt/TOP_LEADERBOARD_0" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1192)